spec/requests/liquid_tags_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe "LiquidTags", type: :request do
+  describe "GET /liquid_tags" do
+    context "when not signed in do" do
+      it "returns a list of all custom Liquid tags" do
+        get liquid_tags_path
+
+        expect(response.status).to eq 401
+      end
+    end
+
+    context "when signed in" do
+      let(:user) { create(:user) }
+      before { sign_in(user) }
+
+      it "returns an array of all custom Liquid tags", :aggregate_failures do
+        get liquid_tags_path
+
+        expect(response.status).to eq 200
+        json = JSON.parse(response.body, symbolize_names: true)
```

Comment:
what about `response.parsed_body` (plus `.with_indifferent_access` if you want to use symbols) ?

Reply:
`HashWithIndifferentAccess` is quite the ugly hack, I changed it to `response.parsed_body` and string keys.